tools/ci/check_stability.py

```diff
@@ -26,11 +27,19 @@
 
 def setup_logging():
     """Set up basic debug logger."""
+    global logger
+    logger = logging.getLogger(wpt_root)
     handler = logging.StreamHandler(sys.stdout)
     formatter = logging.Formatter(logging.BASIC_FORMAT, None)
     handler.setFormatter(formatter)
     logger.addHandler(handler)
     logger.setLevel(logging.DEBUG)
+    logger.propagate = False
```

Comment:
Unsure about this - Added to avoid dup'd messages if someone sets up root-level debugging elsewhere